DigitalLearningSolutions.Web/Views/LearningMenu/PostLearning/PostLearning.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+@model PostLearningAssessmentViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/learningMenu/index.css")" asp-append-version="true">
+
+@{
+  ViewData["HeaderPrefix"] = "";
+  ViewData["Application"] = Model.CourseTitle;
+  ViewData["Title"] = Model.CourseTitle;
+  ViewData["CustomisationId"] = Model.CustomisationId;
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/{Model.CustomisationId}";
+  ViewData["HeaderPathName"] = Model.CourseTitle;
+}
+@section NavMenuItems {
+  <partial name="Shared/_NavMenuItems" />
+}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">
+      <span class="nhsuk-caption-l">Post learning assessment</span>
+      @Model.CourseTitle
+    </h1>
+    <h2 class="nhsuk-u-secondary-text-color">
+      @Model.AssessmentStatus
+    </h2>
+  </div>
+</div>
+
+<div class="nhsuk-inset-text nhsuk-u-margin-top-0 nhsuk-u-margin-bottom-4">
```

Comment:
If this is the same as on the diagnostic assessment then it's probably worth extracting to a partial? 🙂